### DISCUSSION TITLE
Moonlight not using GPU for decoding in Snapdragon 860

### DISCUSSION THREAD

**Discussion by Allekssanderr** [moonlight-mod/moonlight#53] [Q&A] [answered]
Moonlight uses my CPU ( 0 % GPU load) to decode HVEC on HIGH END snapdragon 860 ( Xiaomi Pad 5 Device ) 

![1721735033324_074357](https://github.com/user-attachments/assets/e716f652-9c29-44ae-bfed-5ceb8c21dd25)

 !!! It makes my games laggy and unplayable 
Look at the photo

**Comment by Cynosphere** [comment]
This repo is for a Discord client mod.

If it is related to Discord, upstream Discord/Electron issue. We do not touch anything relating to screenshare.

**Comment by Cynosphere** [comment, marked as answer]
You want to be [here](https://github.com/moonlight-stream/moonlight-qt)
